docx.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.homepage    = 'https://github.com/chrahunt/docx'
   s.files       = Dir["README.md", "LICENSE.md", "lib/**/*.rb"]
 
-  s.add_dependency 'nokogiri', '~> 1.8', '>= 1.8.1'
```

Comment:
As said in the comment https://github.com/ruby-docx/docx/issues/71#issuecomment-560382376 , this `add_dependency` covers `1.10.4` or later. 
If you would like to ensure fixed version of nokogiri must be installed, `'~> 1.10', '>= 1.10.4'` looks better.

Reply:
You're right.  Fixed!